doc/source/create-k8s-cluster.rst

```diff
@@ -163,6 +163,98 @@ Setting up Kubernetes on Microsoft Azure Container Service (ACS)
+--------------------------------------------------
+
+AWS does not have native support for Kubernetes, however there are
+many organizations that have put together their own solutions and
+guides for setting up Kubernetes on AWS. We like the `guide put
+together by Heptio <https://s3.amazonaws.com/quickstart-reference/heptio/latest/doc/heptio-kubernetes-on-the-aws-cloud.pdf>`_,
+and recommend using this for setting up your stack.
+
+1. Follow Step 1 of the Heptio guide. This sets up your Amazon account with
+   the credentials needed to run Kubernetes.
+
+   .. note::
+
+      Make sure that you keep the file downloaded when you create the SSH
+      key. This will be needed later to allow ``kubectl`` to interact with
+      your Kubernetes cluster.
+
+   .. note::
+
+      You may find it helpful to "pin" the services we'll be using to your AWS
+      navbar. This makes it easier to navigate in subsequent sessions.
+      Click the "pin" icon at the top, then drag ``CloudFormation`` and
+      ``EC2`` into your navbar.
+
+2. Follow Step 2 of the Heptio guide. In this section, Heptio allows you
+   to click one of two buttons, each of which run several commands on AWS
+   that set up a "template" of resources we need for Kubernetes.
+   We recommend ``Option 1``, which will create a new set of resources
+   on AWS to run Kubernetes.
+
+   After clicking the link you'll be taken to an AWS page with a field already
+   chosen under "Choose a template". Simply hit "Next". On the following
+   page you'll need to fill in some information. Here are the required fields:
+
+   * ``Stack Name`` can be anything you like.
+   * ``Availability Zone`` is related to the location of the AWS
+     resources. Choose any.
+   * ``Admin Ingress Location`` defines the locations from which you
+     can access this cluster as an administrator. Just enter ``0.0.0.0/0``
+     for the most permissive approach.
+   * ``SSH Key`` is a dropdown list of keys attached to your account.
+     The one you created in Step 1 should be listed here. This will allow
+     you to SSH into the machines if you desire. This will depend on
+     the ``Instance Type`` that you choose. Remember that the most
+     common bottleneck is RAM.
+   * ``Node capacity`` defines the number of machines you've got available.
+   * ``Instance Type`` defines what kind of machine you're requesting. See
+     this `list of instance types with Amazon <https://aws.amazon.com/ec2/instance-types/>`_
+     as well as this list of `pricing for each instance type <https://aws.amazon.com/ec2/pricing/on-demand/>`_.
+   * ``Disk Size`` corresponds to the hard disk for each node. If
+     you need users to keep lots of large files that persist over time,
+     this should be larger.
+   * ``Instance Type (Bastion Host)`` corresponds to a "manager"
+     node that coordinates kubernetes. You can probably leave these as defaults.
+
+   Finally, on the next page you don't need to fill in any of the fields.
```

Comment:
**Enter AWS instance information (page 2):** On the second page, you may leave all of these fields as is or customize. When done, hit ``Next``. Then confirm and ~~his~~ hit...